Cargo.lock

```diff
@@ -667,41 +706,41 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "4.4.6"
+version = "4.5.4"
```

Comment:
@Stebalien I didn't do anything special here to make this bump; probably just a `cargo build`. They don't check in a Cargo.lock so maybe these deps float a bit more than they would otherwise? https://github.com/filecoin-project/rust-filecoin-proofs-api/blob/master/.gitignore#L2 should we fix that?

Reply:
It's also possible that I deleted cargo.lock and had it rebuild because there was some rebasing involved in this PR because the original branch was well behind master. I'll have a play with doing it differently.

Reply:
ahhh I fixed it, it was a rebase problem and was probably from me deleting the lock because of conflicts and rebuilding; I'll have to remember to be more careful about that

Reply:
Yeah, I wish cargo handled that kind of thing better.